docs/user/exams/exam_participation_checker.rst

```diff
@@ -0,0 +1,58 @@
+**************************
+Exam Participation Checker
+**************************
+
```

Comment:
I think the documentation needs some kind of abstract paragraph at the start. Right now the first thing the documentation tells me is to look somewhere else. When someone has never heard of this _thing_ before: Why is telling me to go to exam creation? What is even? What does it do?
From the documentation afterwards it is clear that it is an app (only for Apple devices has to be inferred from the link; only iPad?). Then comes a list of detailed description of specific functions of the app. It is not clear, however, what the high-level goal is those try to achieve.

Just two/three sentences would probably be enough:
- what is it? → it is an iPad app that connects to Artemis
- high-level: what does it do? → instructors can use it to check the participants during a physical exam